### PR TITLE
Enhancement/warning reduction

### PR DESCRIPTION
**Pull request by Kameeno**

### CODE REVIEW

File: src/core/CoolBoardSensors.cpp [hunk 1]
```diff
@@ -57,11 +57,11 @@ void CoolBoardSensors::begin() {
   }
   this->setEnvSensorSettings();
   delay(100);
-  uint8_t envSensorStatus = this->envSensor.begin();
+  this->envSensor.begin();
   // Make sure sensors had enough time to turn on.
   // BME280 requires 2ms to start up.
   delay(1000);
-  DEBUG_VAR("BME280 status after begin is:", envSensorStatus);
+  DEBUG_VAR("BME280 status after begin is:", this->envSensor.begin());
```
Comment (3rwww1) [inline src/core/CoolBoardSensors.cpp]: Ah, I don't think you should call `begin()` twice, sorry... What was the warning here ?

File: src/extras/CoolAdafruit_CCS811.cpp [hunk 2]
```diff
@@ -86,8 +86,8 @@ void Adafruit_CCS811::setEnvironmentalData(uint8_t humidity, double temperature)
 	humidity would be 0x61, 0x00.*/
 	
 	/* Temperature is stored as an unsigned 16 bits integer in 1/512
-	degrees; there is an offset: 0 maps to -25�C. The default value is
-	25�C = 0x64, 0x00. As an example 23.5% temperature would be
+	degrees; there is an offset: 0 maps to -25�C. The default value is
+	25�C = 0x64, 0x00. As an example 23.5% temperature would be
```
Comment (3rwww1) [inline src/extras/CoolAdafruit_CCS811.cpp]: hmmmm those characters are weird ! Are they really responsible for warnings ?

**Review by 3rwww1 — CHANGES_REQUESTED** [review]

File: src/extras/CoolAdafruit_CCS811.cpp [hunk 3]
```diff
@@ -86,8 +86,8 @@ void Adafruit_CCS811::setEnvironmentalData(uint8_t humidity, double temperature)
 	humidity would be 0x61, 0x00.*/
 	
 	/* Temperature is stored as an unsigned 16 bits integer in 1/512
-	degrees; there is an offset: 0 maps to -25�C. The default value is
-	25�C = 0x64, 0x00. As an example 23.5% temperature would be
+	degrees; there is an offset: 0 maps to -25�C. The default value is
```
Comment (trap98) [inline src/extras/CoolAdafruit_CCS811.cpp]: special character

**Review by trap98 — DISMISSED** [review]

File: src/core/CoolBoardSensors.cpp (hunk 1 above)
Comment (Kameeno) [reply src/core/CoolBoardSensors.cpp]: is a warning when you compile in -e prod, 
he say envSensorStatus is a unused variable

File: src/extras/CoolAdafruit_CCS811.cpp (hunk 2 above)
Comment (Kameeno) [reply src/extras/CoolAdafruit_CCS811.cpp]: not, i don't know from where is coming this.

File: src/core/CoolBoard.cpp [hunk 4]
```diff
@@ -658,7 +658,7 @@ void CoolBoard::updateFirmware(String firmwareVersion, String firmwareUrl,
     delete this->coolWifi;
     INFO_LOG("Starting firmware update...");
     t_httpUpdate_return ret =
-        ESPhttpUpdate.update(firmwareUrl, "", firmwareUrlFingerprint, true);
+        ESPhttpUpdate.update(firmwareUrl, "", firmwareUrlFingerprint);
```
Comment (3rwww1) [inline src/core/CoolBoard.cpp:661]: What does it changes ?

File: src/core/CoolBoardSensors.cpp (hunk 1 above)
Comment (3rwww1) [reply src/core/CoolBoardSensors.cpp]: No, we can't merge that and we need to find another solution to fix this. Isn't there a `sensor.getStatus()` method available on the BME class ?

File: src/core/ExternalSensor.h [hunk 5]
```diff
@@ -29,8 +29,8 @@
 #include "CoolAdafruit_TCS34725.h"
 #include "CoolGauge.h"
 #include "CoolNDIR_I2C.h"
-#include "SHT1x.h"
 #include "CoolSDS011.h"
+#include "SHT1x.h"
```
Comment (3rwww1) [inline src/core/ExternalSensor.h:33]: why change the order ?

**Review by 3rwww1 — CHANGES_REQUESTED** [review]

File: src/core/CoolBoard.cpp (hunk 4 above)
Comment (Kameeno) [reply src/core/CoolBoard.cpp:661]: true is a boolean to indicate if need to reboot or not after the ota update, is true for default

**Review by 3rwww1 — APPROVED** [review]

**Pull request merged**